com.unity.shadergraph/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed a bug in master node preview generation that failed compilation when a block was deleted [1319066] (https://issuetracker.unity3d.com/issues/shadergraph-deleting-stack-blocks-of-universal-rp-targeted-shadergraph-causes-the-main-preview-to-fail-to-compile)
 - Fixed a bug where property deduplication was failing and spamming errors [1317809] (https://issuetracker.unity3d.com/issues/console-error-when-adding-a-sample-texture-operator-when-a-sampler-state-property-is-present-in-blackboard)
 - Fixed a bug where synchronously compiling an unencountered shader variant for preview was causing long delays in graph updates [1324429]
+- Fixed a issue when clicking a property in subgraph blackboard will throw null exception errors.[1328377](https://fogbugz.unity3d.com/f/cases/1328377/)
```

Comment:
You need to link to the Issue Tracker, not fb: https://issuetracker.unity3d.com/product/unity/issues/guid/1328377/